examples/kcp-certs/run.bash

```
@@ -0,0 +1,270 @@
+#!/usr/bin/env bash
+
+# cd into repo root
+example_dir="$(dirname "$0")"
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)